server/views/components/actions-card/template.njk

```diff
@@ -20,15 +20,32 @@
         {% for action in params.actions %}
 
           {% if action['render-if'] == null or action['render-if'] === true %}
-            {#- Set classes for this action #}
-            {%- set classNames = "govuk-link" -%}
-            {%- if action.classes %}
-              {% set classNames = classNames + " " + action.classes %}
+            {%- if action.notAUrl %}
```

Comment:
This looks a little complicated, especially where it is being used in the main Overview template (that is particularly hard to read 😬 )

The idea with these components is that they are as generic as we can make them, and have no knowledge of how they are being called. I know you've only used the term "review" in the comment, but the class names (heavy-border, border, no-border) are all from a specific design requirement of the reviews layout.
I think we can do this better .... 🤔